boards/others/canbardo/doc/index.rst

```diff
@@ -0,0 +1,67 @@
+.. zephyr:board:: canbardo
+
+Overview
+********
+
+CANbardo is an open hardware Universal Serial Bus (USB) to Controller Area Network (CAN) adapter
+board. It is designed to be compatible with the open source :ref:`external_module_cannectivity`.
+
+Hardware
+********
+
+The CANbardo board is equipped with an Atmel SAME70N20B microcontroller and features an USB-C
+connector (high-speed USB 2.0), two DB-9M connectors for CAN FD (up to 8 Mbit/s), a number of status
+LEDs, and a push button. Schematics and component placement drawings are available in the `CANbardo
+GitHub repository`_.
+
+Supported Features
+==================
+
+The ``canboardo`` board configuration supports the following hardware features:
```

Comment:
```suggestion
The ``canboardo`` board supports the following hardware features:
```

Reply:
Thank you for reviewing. I've changed the text accordingly.